worker/deploy.sh

```diff
@@ -25,5 +25,9 @@ docker push $TAG
 
 # Update ECS
 ./ecs-deploy -c $CLUSTER -n $SENDING_SERVICE -i $TAG -r $AWS_DEFAULT_REGION --use-latest-task-def -t 600 &
+SENDER_DEPLOY_PID=$!
 ./ecs-deploy -c $CLUSTER -n $LOGGING_SERVICE -i $TAG -r $AWS_DEFAULT_REGION --use-latest-task-def -t 600 &
-wait
+LOGGER_DEPLOY_PID=$!
+
+wait $SENDER_DEPLOY_PID
```

Comment:
@mjgui I'm manually storing the PID for each ECS deploy and waiting on them. Upon failure, the logs print either `"wait $SENDER_DEPLOY_PID" command failed with exit code` or `"wait $LOGGER_DEPLOY_PID" command failed with exit code`.

Reply:
Do we need a more informative error code? If not that should work

Reply:
I think given that the logs from `ecs-deploy` still prints, there should have sufficient information to determine any faults. The error printout at the end will point us to which deploy failed (also we can always checker ECS on AWS console).